stats/simulation.js

```diff
@@ -104,4 +104,11 @@ Simulation.prototype.run = function (outputs) {
   return true
 }
 
+Simulation.prototype.finish = function () {
+  let utxos = this.getUTXOs()
+  this.stats.utxos = utxos.length
+  let costToEmpty = utils.transactionBytes(utxos, []) * this.feeRate // output cost is negligible
```

Comment:
I think you should add the output cost,  because why not. But merging anyway :+1: